src/new-client/src/App/LiveRates/Tile/Header/TileHeader.tsx

```diff
@@ -64,40 +64,50 @@ export const Header: React.FC = () => {
   } = useTileContext()
   const date = useDate(symbol)
   const canTearOut = supportsTearOut
+  const onClick = () => {
+    tearOut(symbol, !isTornOut)
+  }
+  return (
+    <HeaderComponent
+      canTearOut={canTearOut}
+      date={date}
+      base={base}
+      terms={terms}
+      isTornOut={isTornOut}
+      onClick={onClick}
+    />
+  )
+}
 
+export const HeaderComponent: React.FC<{
+  canTearOut?: boolean
+  date: any
+  base: any
+  terms: any
+  isTornOut?: boolean
+  onClick?: () => void
+  tearOutMock?: boolean
+}> = ({
+  canTearOut,
+  base,
+  terms,
+  date,
+  isTornOut,
+  onClick,
+  tearOutMock = false,
+}) => {
   return (
-    <HeaderWrapper>
+    <HeaderWrapper tearOutMock={tearOutMock}>
       <TileSymbol data-qa="tile-header__tile-symbol">
         {base}/{terms}
       </TileSymbol>
       <DeliveryDate data-qa="tile-header__delivery-date">{date}</DeliveryDate>
-      {canTearOut && (
-        <HeaderAction
-          onClick={() => {
-            tearOut(symbol, !isTornOut)
-          }}
-        >
+      {canTearOut && !tearOutMock && (
+        <HeaderAction onClick={onClick ? onClick : () => {}}>
```

Comment:
Minor - Better practice is to pass undefined in cases of no callback to `onX` handlers.
`onClick={onClick || undefined}` or simply `onClick={onClick}` since `onClick` is an optional prop (it will default to `undefined`)